jans-cedarling/flask-sidecar/main/base/cedarling/cedarling.py

```
@@ -52,164 +51,96 @@ def initialize_cedarling(self):
+            payload = token.payload
+            token_input = TokenInput(mapping=mapping, payload=payload)
+            tokens.append(token_input)
+        try:
+            cedarling_request = AuthorizeMultiIssuerRequest(
+                tokens=tokens,
+                action=action_entity,
+                resource=resource_entity,
+                context=context,
+            )
+            authorize_result = self._cedarling.authorize_multi_issuer(cedarling_request)
+        except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)